src/iidm/converter/xml/ShuntCompensatorXml.cpp

```diff
@@ -45,18 +55,39 @@ ShuntCompensator& ShuntCompensatorXml::readRootElementAttributes(ShuntCompensato
 
 void ShuntCompensatorXml::readSubElements(ShuntCompensator& shuntCompensator, NetworkXmlReaderContext& context) const {
     context.getReader().readUntilEndElement(SHUNT, [this, &shuntCompensator, &context]() {
-        AbstractIdentifiableXml::readSubElements(shuntCompensator, context);
+        if (context.getReader().getLocalName() == REGULATING_TERMINAL) {
+            IidmXmlUtil::assertMinimumVersion(SHUNT, REGULATING_TERMINAL, ErrorMessage::NOT_SUPPORTED, IidmXmlVersion::V1_2(), context.getVersion());
+            const auto& id = context.getAnonymizer().deanonymizeString(context.getReader().getAttributeValue(ID));
+            const auto& side = context.getReader().getOptionalAttributeValue(SIDE, "");
+            context.addEndTask([id, side, &shuntCompensator]() {
+                shuntCompensator.setRegulatingTerminal(stdcxx::ref<Terminal>(TerminalRefXml::readTerminalRef(shuntCompensator.getTerminal().getVoltageLevel().getSubstation().getNetwork(), id, side)));
```

Comment:
I think that
`shuntCompensator.getTerminal().getVoltageLevel().getSubstation().getNetwork()`
can be replaced by
`shuntCompensator.getNetwork()`

I also think we fix it in the v1.3

Reply:
Yes this is fixed in `integration/1.3.0`